custom/icds_reports/static/js/angular-services/date-helper.service.js

```diff
@@ -1,6 +1,14 @@
 /* global moment */
 
 window.angular.module('icdsApp').factory('dateHelperService', ['$location', function ($location) {
+    var reportStartDates = {
+        'sdd': new Date(2019, 1),
+    };
+
+    var isSDD =  $location.path().indexOf('service_delivery_dashboard') !== -1;
```

Comment:
is this used? it seems like it's always passed in by calling functions

Reply:
Passed only in functions created in this PR

Reply:
sorry, similar question to the other thread - why does it have to be defined in the service and in the directive?

Reply:
because angular services start only once (i.e. it retains values of its variables even when you switch between components) unlike components, which loses values stored in its variables as soon as we switch to another component. So If you start at SDD, value of isSDD will be true, and will not change even when moved to other reports.